src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,7 +1,33 @@
 package core.basesyntax;
 
+import core.basesyntax.utils.DataUtils;
+
 public class SalaryInfo {
+    static final String FORMAT = "dd.MM.yyyy";
```

Comment:
this field used only in DataUtils, so should be constant of data utils class
```suggestion
    static final String FORMAT = "dd.MM.yyyy";
```

Reply:
moved to DataUtils